cluster-agent/templates/service.yaml

```diff
@@ -4,9 +4,10 @@ metadata:
   name: {{ include "cluster-agent.fullname" . }}
   labels:
     app.kubernetes.io/name: {{ include "cluster-agent.name" . }}
-    helm.sh/chart: {{ include "cluster-agent.chart" . }}
+    helm.sh/chart: {{ include "cluster-agent.name" . }}
```

Comment:
Labels should not include helm chart version as it can create problems during chart upgrades as the chart version will change with upgrade.